index.js

```diff
@@ -54,7 +76,20 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let avgRating = 0;
+  let sumOfRatings = 0;
+  
+  if (movies.length === 0) {
+     avgRating;
```

Comment:
Once again think about how to refactor in the case of objects and falsy values as well.  Also - if the `movie.length < 1` we could just return straight away on 84.